user-interface/src/main/java/life/qbic/datamanager/views/projects/project/access/AddCollaboratorToProjectDialog.java

```diff
@@ -104,7 +104,7 @@ private void initPersonSelection(UserInformationService userInformationService,
         new ComponentRenderer<>(AddCollaboratorToProjectDialog::renderUserInfo));
     personSelection.setRequired(true);
     personSelection.setErrorMessage("Please specify the collaborator to be added to the project");
-    personSelection.setPlaceholder("Please select a username");
+    personSelection.setPlaceholder("Please search for username or full name or ORCiD");
```

Comment:
```suggestion
    personSelection.setPlaceholder("Please search for username or full name or ORCID identifier");
```

its ORCID (https://info.orcid.org/brand-guidelines/)